### ISSUE TITLE
No animation option

### ISSUE THREAD

**Issue by albertdev** [UltimateHackingKeyboard/agent#716]
I have a feature request for a problem similar to #713.

When you switch layers the whole keyboard layout slides to the left / right. While this makes for an impressive sight, I'm getting annoyed that I have to count the number of rows and keys by hand to be sure that I map something in the Fn layer on e.g. the F key. If there were no animation (or a more subtle one) I could just click the base layer, then click the Fn layer and keep my focus on the desired key in the grid to see where I need to click.

TL;DR: a short fading animation between switching layers / keymaps might be more consistent _and_ helpful. The (bolder) sliding effects appear to be better reserved for when you start Agent or saved your configuration to they keyboard.

Hope you can see the reason behind my request, and thanks for all your hard effort!

**Comment by mondalaci** [comment]
I can see the reason behind your request, and appreciate your explanation.

We'll simply add an "Animate layer switching" checkbox under Settings. It'll be enabled by default, but you'll be able to change it, and Agent will remember the setting.

**Comment by mondalaci** [comment]
We'll disable all animations regarding this feature to also address the performance complaints of #823.

**Comment by VinceG3** [comment]
Adding on here that the animation triggering whenever the keyboard changes connected-disconnected state kicks you out of whatever you were doing and returns to the first keymap. At home I use mine connected in my lap so I can't really do anything until I disconnect it. Would really prefer that the agent not be sensitive to the status of the keyboard unless it's actively trying to communicate with it.

**Comment by mondalaci** [comment]
@VinceG3 Thanks for reporting! I created #1091 regarding the mentioned issue. You should be able to simply downgrade Agent until we fix this issue.

**Comment by mondalaci** [comment]
![image](https://user-images.githubusercontent.com/300402/73162365-eb759580-40ed-11ea-9691-7b341f786cba.png)
